packages/ra-input-rich-text/README.md

```diff
@@ -1,79 +1,141 @@
+	<RichTextInput
+		toolbar={
+			<RichTextInputToolbar>
+				<RichTextInputLevelSelect size={size} />
+				<FormatButtons size={size} />
+				<ListButtons size={size} />
+				<LinkButtons size={size} />
+				<QuoteButtons size={size} />
+				<ClearButtons size={size} />
+			</RichTextInputToolbar>
+		}
+		label="Body"
+		source="body"
+		{...props}
+	/>
+);
 ```
 
-**Tip**: When used inside a material-ui `<Card>` (e.g in the default `<Edit>` view), `<RichTextInput>` displays link tooltip as cut off when the user wants to add a hyperlink to a word located on the left side of the input. This is due to an incompatibility between material-ui's `<Card>` component and Quill's positioning algorithm for the link tooltip. 
+## Customizing the editor
 
-To fix this problem, you should override the default card style, as follows:
+You might want to add more Tiptap extensions. The `<RichTextInput>` component accepts an `editorOptions` prop which is the [object passed to Tiptap Editor](https://www.tiptap.dev/guide/configuration).
 
-```diff
-import { Edit, SimpleForm, TextInput } from 'react-admin';
-+import { withStyles } from '@mui/material/styles';
-
--const PostEdit = props => (
-+const PostEdit = withStyles({ card: { overflow: 'initial' } })(props => (
-   <Edit {...props}>
-       <SimpleForm>
-            // ...
-       </SimpleForm>
-   </Edit>
--);
-+));
-```
+If you just want to **add** extensions, don't forget to include those needed by default for our implementation. Here's an example to add the [HorizontalRule node](https://www.tiptap.dev/api/nodes/horizontal-rule):
 
-## License
+```jsx
+import {
+	DefaultEditorOptions,
+	RichTextInput,
+	RichTextInputToolbar,
+	RichTextInputLevelSelect,
+	FormatButtons,
+	AlignmentButtons,
+	ListButtons,
+	LinkButtons,
+	QuoteButtons,
+	ClearButtons,
+} from 'ra-richtext-tiptap';
+import HorizontalRule from '@tiptap/extension-horizontal-rule';
+import Remove from '@material-ui/icons/Remove';
+
+const MyRichTextInput = ({ size, ...props }) => (
+	<RichTextInput
+		editorOptions={MyEditorOptions}
+		toolbar={
+			<RichTextInputToolbar>
+				<RichTextInputLevelSelect size={size} />
+				<FormatButtons size={size} />
+				<AlignmentButtons {size} />
+				<ListButtons size={size} />
+				<LinkButtons size={size} />
+				<QuoteButtons size={size} />
+				<ClearButtons size={size} />
+				<ToggleButton
+					aria-label="Add an horizontal rule"
+					title="Add an horizontal rule"
+					onClick={() => editor.chain().focus().setHorizontalRule().run()}
+					selected={editor && editor.isActive('horizontalRule')}
+				>
+					<Remove fontSize="inherit" />
+			</ToggleButton>
+			</RichTextInputToolbar>
+		}
+		label="Body"
+		source="body"
+		{...props}
+	/>
+);
 
-This library is licensed under the MIT License, and sponsored by [marmelab](https://marmelab.com).
+export const MyEditorOptions = {
+	...DefaultEditorOptions,
+	extensions: [
+		...DefaultEditorOptions.extensions,
+        HorizontalRule,
+	],
+};
+```
```

Comment:
missing license section

Reply:
Added. We have several packages without this section btw